cache/stringcache/string_caches.go

```diff
@@ -50,7 +50,11 @@ func (cache stringMap) contains(searchString string) bool {
 	})
 
 	if idx < searchBucketLen {
-		return cache[searchLen][idx*searchLen:idx*searchLen+searchLen] == strings.ToLower(normalized)
+		blockRule := cache[searchLen][idx*searchLen : idx*searchLen+searchLen]
+		if blockRule == normalized {
+			log.PrefixedLog("stringMap").Debugf("block rule '%s' matched with '%s'", blockRule, searchString)
```

Comment:
We usually have the prefixes/tags be `snake_case`:
```suggestion
			log.PrefixedLog("string_map").Debugf("block rule '%s' matched with '%s'", blockRule, searchString)
```

Reply:
That was copy-pasted from
```
log.PrefixedLog("regexCache")
```

Reply:
Could you change that one too then please?

The examples of snake_case I'm thinking of are mostly in the resolvers, for instance:
https://github.com/0xERR0R/blocky/blob/v0.23/resolver/fqdn_only_resolver.go#L22
https://github.com/0xERR0R/blocky/blob/v0.23/resolver/query_logging_resolver.go#L18

Reply:
Sure, I've already done this in [8eff14f](https://github.com/0xERR0R/blocky/pull/1460/commits/8eff14fee0f98ab23b5b8bf0d9c177f999f55337) :) I just tried to explain why it was like this :)

Reply:
Ah sorry missed it!